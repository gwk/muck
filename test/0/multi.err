muck note: a-to-many-0.txt: building: `python3.6 'a-to-many-{}.txt.py' 0`
muck note: a-to-many-0.txt: finished: (via manifest).
muck note: a-to-many-0.txt: product changed; 12 B.
muck note: a-to-many-1.txt: product changed; 12 B.
muck note: c-many.txt: building: `python3.6 c-many.txt.py`
muck note: c-many.txt: finished: (via stdout).
muck note: c-many.txt: product changed; 0 B.
