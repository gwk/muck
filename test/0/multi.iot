{
  'cmd': '$MUCK -no-times c-many.txt',
  'code': 0,
  'links': {
    'test/0/multi/a-to-many-%.txt.py',
    'test/0/multi/b-through-%.txt.py',
    'test/0/multi/c-many.txt.py',
  },
  'files': {
    '_build/c-many.txt' : {
      'path': '$STEM/c-many-exp.txt',
    }
  }
}
