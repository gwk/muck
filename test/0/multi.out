
a-to-many-0.txt (dependents: b-through-0.txt c-many.txt.py):
  a-to-many-{i}.txt.py

a-to-many-1.txt (dependents: b-through-1.txt c-many.txt.py):
  a-to-many-{i}.txt.py

b-through-{i}.txt.py (dependents: b-through-0.txt b-through-1.txt):
 ~a-to-many-{i}.txt

c-many.txt:
  c-many.txt.py:
    a-to-many-0.txt*
    a-to-many-1.txt*
    b-through-0.txt:
      b-through-{i}.txt.py*
      a-to-many-0.txt*
    b-through-1.txt:
      b-through-{i}.txt.py*
      a-to-many-1.txt*
